velox/docs/functions/spark/datetime.rst

```diff
@@ -22,6 +22,12 @@ These functions support TIMESTAMP and DATE input types.
 
     num_days can be positive or negative.
 
+.. spark:function:: dayofweek(date) -> integer
```

Comment:
`dayofweek(date)` -> `dayofweek(x)`, mark x could be data and timestamp.
```
.. function:: date_format(x, format) -> varchar

    Formats ``x`` as a string using ``format``. ``x`` is a timestamp or a timestamp with time zone.
```

Reply:
Fixed